src/components/formBuilder/formBuilder.jsx

```diff
@@ -0,0 +1,280 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import { saveFormSchemaThunk } from '../../redux/forms-Reducer';
+import style from './formBuilder.module.css';
+
+function FormBuilder(props) {
+  const [state, setState] = useState({
+    inputs: [],
+    formName: '',
+    formDesc: ''
+  });
+  const [active, setActive] = useState('formName');
```

Comment:
```suggestion
  const [active, setActive] = useState('formName');
```
this formName is confusing because it is also present in state.formName above.
Can you explain what you wanted to achieve?